versioned_docs/version-2.x/advanced/payments/payment-on-account.mdx

```diff
@@ -28,6 +28,17 @@ This feature is directly provided by Adobe Commerce. Please refer to
 
 :::
 
+In addition, this feature is also controlled by the roles and the associated
+permissions assigned to a user within the company. For now, those roles can only
+be set through the Magento frontend. So to make sure a company user can use the
```

Comment:
**issue:** It isn't viable to suggest to proceed like this for production use cases.

**suggestion:** mention programatical  roles management for production, and highlight that the procedure is for local developments only (or dev)

```suggestion
be set programmtically or through the Magento frontend. In local environments, to make sure a company user can use the
```